src/algebra/group/units.lean

```diff
@@ -120,6 +120,9 @@ section monoid
   @[simp] theorem divp_right_inj (u : units α) {a b : α} : a /ₚ u = b /ₚ u ↔ a = b :=
   units.mul_right_inj _
 
+  @[simp] theorem divp_divp_eq (x : α) (u₁ u₂ : units α) : (x /ₚ u₁) /ₚ u₂ = x /ₚ (u₂ * u₁) :=
```

Comment:
I suggest the name `divp_divp_eq_divp_mul` which matches `field.div_div_eq_div_mul` from core.
Also, should this really be a simp lemma? It's not clear to me which side is simpler.

Reply:
Renamed, removed `@[simp]`.